tracing-core/src/metadata.rs

```diff
@@ -91,18 +93,149 @@ pub struct Metadata<'a> {
 pub struct Kind(KindInner);
 
 /// Describes the level of verbosity of a span or event.
+///
+/// # Comparing Levels
+///
+/// `Level` implements the [`PartialOrd`] and [`Ord`] traits, allowing two
+/// `Level`s to be compared to determine which is considered more or less
+/// verbose. Levels which are more verbose are considered "greater than" levels
+/// which are less verbose, with [`Level::ERROR`] considered the lowest, and
+/// [`Level::TRACE`] considered the highest.
+///
+/// For example:
+/// ```
+/// use tracing_core::Level;
+///
+/// assert!(Level::TRACE > Level::DEBUG);
+/// assert!(Level::ERROR < Level::WARN);
+/// assert!(Level::INFO <= Level::DEBUG);
+/// assert_eq!(Level::TRACE, Level::TRACE);
+/// ```
+///
+/// # Filtering
+///
+/// Typically, `Level`s are used to implement *filtering* that determines which
+/// spans and events are enabled. Depending on the use case, more or less
+/// verbose diagnostics may be desired &mdash; for example, when running in
+/// development, [`DEBUG`]-level traces may be enabled by default, and when
+/// running in production, we might choose to enable only [`INFO`]-level and
+/// lower. Libraries may include very verbose diagnostics at the [`DEBUG`]
+/// and/or [`TRACE`] levels, which applications using those libraries might
+/// typically choose to ignore. However, when debugging an issue involving those
+/// libraries, it may be useful to temporarily enable those traces.
```

Comment:
```suggestion
/// development, [`DEBUG`]-level traces may be enabled by default. When running in
/// production, only [`INFO`]-level and lower traces might be enabled. Libraries 
/// may include very verbose diagnostics at the [`DEBUG`] and/or [`TRACE`] levels.
/// Applications using those libraries typically chose to ignore those traces. However, when
/// debugging an issue involving said libraries, it may be useful to temporarily 
/// enable the more verbose traces.
```

Removing the we—it wasn't used before.